app/flatpak-builtins-info.c

```diff
@@ -406,12 +406,10 @@ flatpak_builtin_info (int argc, char **argv, GCancellable *cancellable, GError *
 
       if (opt_show_metadata)
         {
-          g_autoptr(GFile) deploy_dir = NULL;
           g_autoptr(GFile) file = NULL;
           g_autofree char *data = NULL;
           gsize data_size;
 
-          deploy_dir = flatpak_dir_get_if_deployed (dir, ref, NULL, cancellable);
```

Comment:
Actually, this seems deploy_dir seems more right than the toplevel one which is:
```
  deploy = flatpak_find_deploy_for_ref (ref, NULL, NULL, error);
```
Which means it could come from a system dir even if you said --user. That doesn't seem right.

Reply:
Yeah, added a separate commit to fix that